WalletWasabi.Fluent/Behaviors/TextBoxAutoSelectTextBehavior.cs

```diff
@@ -9,11 +9,11 @@ internal class TextBoxAutoSelectTextBehavior : AttachedToVisualTreeBehavior<Text
 	protected override void OnAttachedToVisualTree(CompositeDisposable disposable)
 	{
 		AssociatedObject?.SelectAll();
-		Observable.FromEventPattern(AssociatedObject, "GotFocus")
-			.Subscribe(_ => AssociatedObject.SelectAll())
+		Observable.FromEventPattern(AssociatedObject!, "GotFocus")
+			.Subscribe(_ => AssociatedObject!.SelectAll())
 			.DisposeWith(disposable);
-		Observable.FromEventPattern(AssociatedObject, "PointerReleased")
-			.Subscribe(_ => AssociatedObject.SelectAll())
+		Observable.FromEventPattern(AssociatedObject!, "PointerReleased")
+			.Subscribe(_ => AssociatedObject!.SelectAll())
```

Comment:
No, this is bad. The warning is there because it can be null. `!` just suppresses the warning.
Rather check for null at the beginning, that is the usual approach in other behaviors.
```
if (AssociatedObject is null)
{
	return;
}
```

Reply:
> No, this is bad. The warning is there because it can be null. `!` just suppresses the warning. Rather check for null at the beginning, that is the usual approach in other behaviors.
> 
> ```
> if (AssociatedObject is null)
> {
> 	return;
> }
> ```

I think I was drunk...